localization/languages/pt-BR.json

```diff
@@ -122,68 +122,66 @@
+        "settingsContentBlockingLevel2": "Bloquear todos os anúncios e rastreadores",
+        "settingsContentBlockingExceptions": "Permitir anúncios nos sites:",
         "settingsBlockScriptsToggle": "Bloquear scripts",
         "settingsBlockImagesToggle": "Bloquear imagens",
-        "settingsEasyListCredit": {
-            "unsafeHTML": null //missing translation
-        },
+        "settingsEasyListCredit": {"unsafeHTML": "Com base na EasyList e EasyPrivacy, criadas por <a target=\"_blank\" href=\"https://easylist.to/\">The EasyList authors</a> (<a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">view license</a>)."},
         "settingsAppearanceHeading": "Aparência",
-        "settingsDarkModeToggle": "Habilitar modo noturno",
-        "settingsSiteThemeToggle": null, //missing translation
+        "settingsDarkModeToggle": "Habilitar modo darck",
+        "settingsSiteThemeToggle": "Habilitar mudanças de cores na barra de endereços",
         "settingsHistoryButtonToggle": "Habilitar botão voltar",
-        "settingsAdditionalFeaturesHeading": null, //missing translation
-        "settingsUserscriptsToggle": null, //missing translation
-        "settingsUserscriptsExplanation": null, //missing translation
-        "settingsUpdateNotificationsToggle": null, //missing translation
+        "settingsAdditionalFeaturesHeading": "Recursos adicionais",
+        "settingsUserscriptsToggle": "Habilitar scripts do usuário",
+        "settingsUserscriptsExplanation": {"unsafeHTML":"Scripts do usuário permitem modificar o comportamento do site - <a href=\"https://github.com/minbrowser/min/wiki/userscripts\">Saiba mais</a>."},
+	"settingsUpdateNotificationsToggle": "Notificações de atualização",
         "settingsSearchEngineHeading": "Serviço de busca",
         "settingsDefaultSearchEngine": "Escolha o serviço de busca padrão:",
         "settingsDDGExplanation": "Defina DuckDuckGo como o serviço padrão de buscas para ver respostas instantâneas na barra de busca.",
-        "customSearchEngineDescription": null, //missing translation
+        "customSearchEngineDescription":"Substituir o serviço de pesquisa por %s",
         "settingsKeyboardShortcutsHeading": "Teclas de atalho",
-        "settingsKeyboardShortcutsHelp": null, //missing translation
+        "settingsKeyboardShortcutsHelp":"Use vírgulas para separar vários atalhos.",
         /* app menu */
         "appMenuFile": "Arquivo",
-        "appMenuNewTab": "Nova aba",
-        "appMenuDuplicateTab": null, //missing translation
-        "appMenuNewPrivateTab": "Nova aba privada",
+        "appMenuNewTab": "Nova guia",
+        "appMenuDuplicateTab": "Duplicar guia atual",
+        "appMenuNewPrivateTab": "Nova guia privada",
         "appMenuNewTask": "Nova tarefa",
-        "appMenuSavePageAs": "Salvar página como",
-        "appMenuPrint": "Imprimir",
+        "appMenuSavePageAs": "Salvar página como...",
+        "appMenuPrint": "Imprimir...",
```

Comment:
Please leave the ellipses out here.

It looks like the standard for macOS at least is to have ellipses for these actions, which I hadn't noticed before, but it's probably better to append those in the actual menu code rather than the translation file.

Reply:
ok, mas devo removê-lo ou não? kkk

Reply:
ok, but should i remove it or not? kkk

Reply:
If it's needed for the text to be gramatically correct in Portuguese, leave it, otherwise please remove it.

Reply:
the "..." can be used to intentionally omit something you should or could say.

Print page -> Print ...
Save page with file-> save page as ...

Reply:
If it's just a style difference, I'd prefer to leave them out for now, but if it's needed for the sentence to be grammatically correct then I'd be fine with leaving them.

Is just "Imprimir" something that would be gramatically correct to say?

Reply:
in the case of "Print", the "..." can be removed without impact, in "save page as (what?)" if you look, words are missing, the impact will be greater, but the removal would not be wrong either, this is an educated norm, but I noticed that several browsers: chrome, firefox, opera, vivalde and others use -> "print ...",
"save page as ...", "search ...",
"clean navigation data..."

Reply:
OK. Can you remove them from the translation file then, and I'll update the menu bar code to add them regardless of the language?